packages/near-ether/src/bridged-ether/sendToEthereum/index.ts

```diff
@@ -704,9 +702,13 @@ export async function checkSync (
     options.nearAccount?.connection.provider ??
     getNearProvider()
   if (nearOnEthClientBlockHeight > withdrawBlockHeight) {
+    const etherNep141FactoryMigrationHeight = options.etherNep141FactoryMigrationHeight ?? bridgeParams.etherNep141FactoryMigrationHeight
+    const etherNep141Factory = withdrawBlockHeight >= etherNep141FactoryMigrationHeight
+      ? (options.etherNep141Factory ?? bridgeParams.etherNep141Factory)
+      : bridgeParams.auroraEvmAccount
```

Comment:
This code is duplicated many times. Can it be extracted?

Reply:
Yea good idea. https://github.com/aurora-is-near/rainbow-bridge-client/pull/98/commits/a49efb543ac2aa0ea430a994b940bdf50181e4cd